app/src/main/res/values/strings.xml

```diff
@@ -9,6 +9,7 @@
     <string name="save">Save</string>
     <string name="send">Send</string>
     <string name="id">Id</string>
+    <string name="parcel_id">Id (parcel)</string>
```

Comment:
With this new field, the UI looks quite odd. I think we should rename:

- "Id" -> "Ping id".
- "Id (parcel)" -> "Parcel id".